tests/e2e/machine/rest.go

```diff
@@ -64,10 +64,10 @@ func (s *RestE2ETestSuite) TestAttestMachineREST() {
 	// Register TA
 	ta := sample.TrustAnchor(pubKey)
 	taMsg := machinetypes.MsgRegisterTrustAnchor{
-		Creator:     addr.String(),
+		Creator:     val.Address.String(),
```

Comment:
before introducing the `lib` here it used to be `addr.String()` too...

Reply:
true, but with this change trust anchor registration is permitted only for validators until we have a process for registering 3rd party OEMs